Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1046,6 +1046,10 @@ public virtual void ProcessSynchronousEvents(bool forceProcess = false)
                 //Set next sample time: 4000 samples per backtest
                 _nextSample = time.Add(ResamplePeriod);
 
+                // Check to see if we should update stored portfolio and bench values
+                UpdatePortfolioValue(time, forceProcess);
+                UpdateBenchmarkValue(time, forceProcess);
```

Comment:
Cool!
Careful with race condition with the realtime schedule event, we will have to use `ReferenceWrapper` for these values I think, because of `decimal` not being thread safe
Would be awesome to add some tests for live trading asserting the behavior too 💯